front/app/containers/Admin/settings/customize/index.tsx

```diff
@@ -782,21 +795,15 @@ class SettingsCustomizeTab extends PureComponent<
                     </LabelContent>
                   </ToggleLabel>
                 </Setting>
-              </WideSectionField>
-
-              {tenant.data.attributes.settings?.events_widget?.allowed && (
```

Comment:
This line was the reason I started this PR. Outlets should always render. The conditions should happen on a component basis in the module's index file.